### PR TITLE
docs(problem-solving):include problem solving article to ReadME

### LINKED ISSUE #347: [Suggestion] Problem solving for developers
Before seeking help else where, developers should learn the much needed skill of problem solving. I would like to suggest adding  a article to the, "_Need Help_", section of the standards and practice repo that breaks down how to problem solve in a general way.

### PR DESCRIPTION
**Pull request by hstamps-stack**
## Changes

1. Added a sub section to the Need help section about problem solving
2. Included an article about problem solving from freecodecamp.

## Purpose
Help developers better problem solve before asking help on tasked issues.
## Approach
The freecodecamp article gives a general guideline on problem solving

https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/

Closes #347

### CODE REVIEW

**Review by jeffreyriosjr — APPROVED** [review]
brilliant!

**Review by meru20 — APPROVED** [review]
Helpful tips!🙌🏾

File: README.md [hunk 1]
```diff
@@ -117,6 +117,12 @@ Getting stuck is a natural part of development and happens to the best of us. It
 1. **Your team channel** - When you start a project, you will be added to a private team channel. Feel free to ask your questions there! Your team lead may be able to help you through your blocker.
 2. **[bwtc-stackoverflow](https://stackoverflow.com/c/bwtc/questions)** - Did you know we have our own Stack Overflow? Take a look, it has some great quesions and answers. 
 3. **[bwtc-technical-discussions](https://bitwiseindustries.slack.com/archives/CN462BE5D)** - Great place for technical questions that involve some code.
+
+#### Problem Solving
```
Comment (B-cavazos) [inline README.md:121]: I think this section would make more sense if it were placed before Communication Channels. Reading the Problem Solving section first reinforces the idea that we should make an attempt on the problem before reaching out for help.

**Review by ludgranados — APPROVED** [review]
I agree with B-cavazos regarding Communication Channels having a clearer understanding when they follow the Problem Solving section.

**Review by michaelachrisco — APPROVED** [review]
Oee comment and potential refactor. Im ok with linking the page for now, but I believe we should potentially make a new page before merging.

File: README.md [hunk 2]
```diff
@@ -117,6 +117,12 @@ Getting stuck is a natural part of development and happens to the best of us. It
...
+While reaching out for help is very important when getting stuck, developers are expected to try and problem solve before seeking help else where. A good read on how to problem solve is listed below
+
+[Problem Solving](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/)
```
[the remark this answers is not in the document]
Comment (hstamps-stack) [reply README.md:124]: Yeah, I will look into creating a page. Thanks for the recommendation.

[the remark this answers is not in the document]
Comment (michaelachrisco) [reply README.md:124]: If you would like, you can create a separate Github issue addressing the above (separate problem solving page) and ill go ahead and merge this PR.

[the remark this answers is not in the document]
Comment (hstamps-stack) [reply README.md:124]: Sounds good to me

**Pull request merged**